src/azure-cli/azure/cli/command_modules/backup/_help.py

```diff
@@ -26,6 +26,22 @@
     crafted: true
 """
 
+helps['backup container register'] = """
+type: command
+short-summary: This command allows Azure Backup to register a Resource to the given Recovery Services Vault.
```

Comment:
Please refine your short summary and the following examples name using active voice and brief message.
e.g. Register a Resource to the given Recovery Services Vault.

Reply:
Agreed